doc/redirects.md

```diff
@@ -0,0 +1,53 @@
+# Redirects
+
+Sometimes (docs) pages get removed, renamed, and added. There is [a redirects file](../www/_data/redirects.yml) that contains redirects for such occasions. It has three sections:
+
+- `general`: single-page redirects.
+- `docs-global`: redirects across all docs versions and languages,
+- `docs`: version-specific docs redirects, and
+
+For non-docs URIs, there are no versioning considerations. Make redirects like so:
+
+    general:
+        - {old: "old/uri/for/page.html", new: "its/new/uri.html"}
+
+**NOTE**: Review (and test, if possible) these redirects before making them live, since they're permanent (HTTP 301) redirects. Incorrect permanent redirects will make a URI almost impossible to bring back into browsers and search indices.
+
+## Changing Docs URIs
```

Comment:
Why is this a new section?

Reply:
I felt the list of cases below required a "parent" headline. Is it factually wrong? Or should the `general` above also get one?

Reply:
It's not wrong. It was just all one section before, so I'm wondering why you split it into two sections. Both are OK.